### PR TITLE
chore: Added Tailwind CSS

### PR DESCRIPTION
**Pull request by BartoszJarocki**
# Description

This PR adds [Tailwind CSS](https://tailwindui.com) as an alternative for building UI with `emotion`. 
Docs are available [🔒 here.](https://www.notion.so/parabol/Tailwind-CSS-d8720612547740c297b9799aebcc7666)

TODO:
- [x] check prod build
- [x] make sure prod bundle has only used tailwind classes
- [x] add prettier plugin to sort tailwind classes
- [x] configure color palette in tailwind config
- [x] configure breakpoints
- [x] add VSCode extension recommendation in /.vscode/extensions (from Matt)
- [x] add an example, common gotcha's and best practices 

## Demo

localhost_3000_meetings

## Testing scenarios

- [ ] check dev environment 
- [ ] check prod environment

## Final checklist

- [ ] I checked the [code review guidelines](../docs/codeReview.md)
- [ ] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [ ] I have performed a self-review of my code, the same way I'd do it for any other team member
- [ ] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [ ] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [ ] PR title is human readable and could be used in changelog

### CODE REVIEW

**Review by mattkrick — APPROVED** [review]

File: tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  corePlugins: {
+    preflight: false
+  },
+  content: ['./packages/client/**/*.{ts,tsx,js,jsx,html}'],
+  theme: {
+    fontFamily: {
+      sans: ['IBM Plex Sans', ...defaultTheme.fontFamily.sans],
+      mono: ['IBM Plex Mono', ...defaultTheme.fontFamily.mono]
+    },
+    colors: {
```
[the remark this answers is not in the document]
Comment (BartoszJarocki) [reply tailwind.config.js:19]: I'm not sure if this is possible, but we can have presets in `tailwind.palette.js` or something like that, import it in `tailwind.config.js` and also, import it in the code so we can have a single source of truth. This way there should be no problems with tree-shaking, wdyt?

File: scripts/webpack/prod.client.config.js [hunk 2]
```diff
@@ -213,7 +213,7 @@ module.exports = ({isDeploy, isStats}) => ({
       {test: /\.flow$/, loader: 'ignore-loader'},
       {
         test: /\.css$/,
-        use: ['style-loader', 'css-loader']
+        use: ['style-loader', 'css-loader', 'postcss-loader']
```
[the remark this answers is not in the document]
Comment (BartoszJarocki) [reply scripts/webpack/prod.client.config.js:216]: I added but seems like it needs a little more time to get it right, I'm getting
```
  ModuleParseError: Module parse failed: Unexpected character '@' (1:0)
  You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
  > @tailwind base;
  | @tailwind components;
  | @tailwind utilities;
```
are you ok with shipping this PR as is and adding `MiniCssExtractPlugin` in a separate PR?

**Comment by BartoszJarocki** [comment]
@mattkrick let's not merge this yet, all of the sudden I get some webpack errors and app isn't running, will investigate

**Comment by BartoszJarocki** [comment]
@mattkrick ok, seems like we had some updates and now, all tailwind related packages have to be in installed in the root, otherwise webpack won't see a thing. would you mind checking if everything works locally on your end?

**Comment by mattkrick** [comment]
i was wondering about that... i also started putting all build-time packages as dev dependencies in the root package.json for the same reason. 

they usually work themselves out if you `yarn` from root instead of `yarn` from packages/client, but always seems like there's a little weirdness to it

File: tailwind.config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (mattkrick) [reply tailwind.config.js:19]: i like it!

File: scripts/webpack/prod.client.config.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (mattkrick) [reply scripts/webpack/prod.client.config.js:216]: for sure, safe to merge without, i'll make a quick lil follow up PR

[the remark this answers is not in the document]
Comment (BartoszJarocki) [reply scripts/webpack/prod.client.config.js:216]: here's a commit on another branch where I tried to use it: https://github.com/ParabolInc/parabol/commit/683cc117d027dcf9d4e99312db69d74652b423ad 

currently fighting with a lot of that kind of errors:
```
Error: Conflict: Multiple assets emit different content to the same filename ArchiveTaskUserRoot_8cbc38c08a0025b9aa02.js.ma
```

**Comment by mattkrick** [comment]
LGTM! You can merge now or extract the colors out & then merge, up to you

**Pull request merged**